Source/WebCore/css/StyleProperties.cpp

```diff
@@ -151,14 +151,9 @@ String StyleProperties::commonShorthandChecks(const StylePropertyShorthand& shor
     std::optional<bool> importance;
     size_t numSetFromShorthand = 0;
     for (auto longhand : shorthand) {
-        // FIXME: Shouldn't serialize the shorthand if some longhand is missing.
```

Comment:
I think it would be good to try to split the patch into smaller ones, for example I believe this fix could be done first by itself.

Reply:
Sure I’d be willing to split into smaller ones. The biggest job is rebasing all the regression tests, which makes splitting it up a long process; that’s what’s taken me so long here. I am a little worried that some of the changes break things that are then fixed by other changes, so splitting may be a challenging project.

Reply:
I have greatly reduced the patch.